docs/components/early-access/alpha/mcp-client/mcp-client.md

```diff
@@ -5,218 +5,55 @@ sidebar_label: MCP Client
-
-Select the operation to perform:
-
-| Operation  | Description                                              | Configuration                                                                                                  |
-| :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| List tools | Retrieves the list of tools available on the MCP server. | No additional configuration required.                                                                          |
-| Call tool  | Invokes a specific tool on the MCP server.               | **Tool name**: The name of the tool to invoke.<br/>**Arguments**: Tool arguments as a FEEL context expression. |
-
-#### Tools
-
-Allows filtering the list of tools provided by the MCP server. If not configured, all tools provided by the MCP server will be available to the AI agent.
-
-| Field          | Required | Description                                             | Example                                |
-| :------------- | :------- | :------------------------------------------------------ | :------------------------------------- |
-| Included tools | No       | List of allowed tools provided by the MCP server.       | `["read_file", "read_multiple_files"]` |
-| Excluded tools | No       | List of tools to exclude. Overrides any included tools. | `["write_file"]`                       |
-
-For example, an MCP client connected to
-a [file system MCP server](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem) could be configured
-with `["read_file", "read_multiple_files"]` as included tools to only allow read-only operations to the file system.
-Alternatively, it could be configured with `["write_file"]` as a list of excluded tools to prevent write operations.
-
-#### Operation
-
-Configures the operation to execute on the MCP server. You typically only need to change the default value if the ad-hoc sub-process multi-instance uses an input element other than `toolCall`.
-
-| Field      | Required | Description                                                                                                                                       |
-| :--------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Method     | Yes      | The [MCP method](https://modelcontextprotocol.io/specification/2025-06-18/server/tools#protocol-messages) to call. Defaults to `toolCall.method`. |
-| Parameters | Yes      | The parameters to pass with the MCP client execution. Defaults to `toolCall.params`.                                                              |
-
-#### Output mapping
-
-Specify the process variables to map and export the tool calling response into.
+:::tip
+Camunda recommends the Streamable HTTP transport protocol over HTTP with SSE.
```

Comment:
Typically, this is not really a choice, but rather what a provider implements. The SSE one is deprecated and was removed from the spec. We as Camunda don't really recommend either, but we might drop support for SSE at some point.

Maybe we don't really need to make recommendation here, but could just mark the SSE one as legacy/deprecated?